infra/cmake/packages/Pybind11Config.cmake

```diff
@@ -0,0 +1,14 @@
+function(_Pybind11_import)
+  nnas_find_package(Pybind11Source QUIET)
+
+  if(NOT Pybind11Source_FOUND)
+    set(Pybind11_FOUND FALSE PARENT_SCOPE)
+    return()
+  endif(NOT Pybind11Source_FOUND)
+
+  add_subdirectory(${Pybind11Source_DIR} ${CMAKE_CURRENT_BINARY_DIR}/pybind11)
```

Comment:
I'm not sure about the convention over this ``), but ``CMAKE_CURRENT_BINARY_DIR`` may be dangerous ;-)

Reply:
I'm not very familiar with cmake. Can you explain a little bit more about the dangerous cases? It will be helpful for me to avoid such cases.

Reply:
This will create a pybind11 directory for for the first ``find_pacakge`` call is made. There is no guarantee that there is no duplication ;-D

Reply:
@jinevening You can refer `infra/cmake/packages/GTestConfig.cmake` or `infra/cmake/packages/FlatBuffersConfig.cmake` for external library build.